File: Python/Condicional/Intermedio/Ejercicio_03.py
import random

# Generar número aleatorio para el descuento
descuento_aleatorio = random.randint(1, 100)

# Solicitar al usuario ingresar el total de la compra
total_compra = float(input("Ingrese el total de la compra: "))

# Calcular el descuento según la regla dada
if descuento_aleatorio < 74:
    descuento = total_compra * 0.15
else:
    descuento = total_compra * 0.20

print("El descuento es:", descuento)

File: Java/src/main/java/org/sisuns/condicionales/intermedio/Ejercicio_03.java
package org.sisuns.condicionales.intermedio;

public class Ejercicio_03 {
    public static void main(String[] args) {
        // Asignar un valor específico al descuento aleatorio (por ejemplo, 50)
        double descuentoAleatorio = 50;

        // Definir el total de la compra
        double totalCompra = 0; // Puedes asignar un valor específico si lo deseas

        // Calcular el descuento según la regla dada
        double descuento;
        if (descuentoAleatorio < 74) {
            descuento = totalCompra * 0.15;
        } else {
            descuento = totalCompra * 0.20;
        }

        // Mostrar el descuento
        System.out.println("El descuento es: " + descuento);
    }
}
